src/Auth0.php

```diff
@@ -425,6 +461,46 @@ public function getUser(): ?array
         return $this->getState()->getUser();
     }
 
+    public function handleBackchannelLogout(
+        string $logoutToken,
+    ): TokenInterface {
+        $cache = $this->configuration()->getBackchannelLogoutCache();
+
+        // The SDK must be configured for authentication (statefulness) to invoke this method.
+        if (! $this->configuration()->usingStatefulness() || ! $cache instanceof \Psr\Cache\CacheItemPoolInterface) {
+            throw ConfigurationException::requiresStatefulness('Auth0->handleBackchannelLogout()');
+        }
+
+        // Decode the logout token. If this ste fails, an exception will be thrown.
+        $token = $this->decode(
+            token: $logoutToken,
+            tokenType: \Auth0\SDK\Token::TYPE_LOGOUT_TOKEN,
+        );
+
+        // Create a reference key for comparison against future requests.
+        $backchannel = hash('sha256', implode('|', [$token->getSubject() ?? '', $token->getIssuer() ?? '', $token->getIdentifier() ?? '']));
+
+        // Retrieve any existing reference, or silently create a new one.
+        $request = $cache->getItem($backchannel);
+
+        // Setup the backchannel logout request record:
+        $request->set(json_encode([
+            'sub' => $token->getSubject(),
+            'iss' => $token->getIssuer(),
+            'sid' => $token->getIdentifier(),
+            'iat' => $token->getIssued(),
+        ]));
+
+        // Let the backchannel logout request fall off after a reasonable amount of time.
+        $request->expiresAfter(time() + (86400 * 30));
```

Comment:
Do you have a session duration setting you can set this to? (shortest of rolling or absolute duration)

Reply:
👍 Updated to make this configurable; a new option is exposed, `backchannelLogoutExpires.` It still defaults to 30 days to cover some edge cases I can foresee with rolling sessions under certain circumstances.